README.md

```diff
@@ -5,32 +5,32 @@ PCSX2 is a free and open-source PlayStation 2 (PS2) emulator. Its purpose is to
 
 # Project Details
 
-The PCSX2 project has been running for more than ten years. Past versions could only run a few public domain game demos, but newer versions can run many games at full speed, including popular titles such as Final Fantasy X and Devil May Cry 3. Visit the [PCSX2 homepage](https://pcsx2.net) to check the latest compatibility status of games (with more than 2000 titles tested), or ask for help in the [official forums](https://forums.pcsx2.net/).
+The PCSX2 project has been running for more than ten years. Past versions could only run a few public domain game demos, but newer versions can run most games at full speed, including popular titles such as Final Fantasy X and Devil May Cry 3. Visit the [PCSX2 compatibility list](https://pcsx2.net/compatibility-list.html) to check the latest compatibility status of games (with more than 2000 titles tested), or ask for help in the [official forums](https://forums.pcsx2.net/).
 
 The latest officially released stable version is version 1.4.0.
 
 Installers and binaries for both Windows and Linux are available from [our website](https://pcsx2.net/download.html).
 
-Development builds are also available from [our website](https://pcsx2.net/download/development/git.html).
+Development builds are also available from [our website](https://pcsx2.net/download/development.html).
 
 # System Requirements
 
-## Minimum
-* OS: Windows 7 or GNU/Linux
-* CPU: Any that supports SSE2 @ [1600 STR](#Notes)
-* GPU: DirectX 10 support
-* RAM: 2GB
+#### Minimum
 
-## Recommended
-* OS: Windows 10 (64-bit) or GNU/Linux (64-bit)
-* CPU: Any that supports AVX2 (Core series Haswell or Ryzen and newer) @ [2000 STR](#Notes) or better
-* GPU: DirectX 11 support or better
-* RAM: 4GB or more
+| Operating System | CPU | GPU | RAM |
+| --- | --- | --- | --- |
+| - Windows 8.1 or newer (32 or 64 bit) <br/> - Ubuntu/Debian 18.04 or newer, Arch Linux, or other distro (32 or 64 bit) | - Supports SSE2 <br/> - [PassMark Single Thread Performance](https://www.cpubenchmark.net/singleThread.html) rating near or greater than 1650 <br/> - Two physical cores, with hyperthreading | - Direct3D11 support <br/> - OpenGL 3.x support <br/> - [PassMark G3D Mark](https://www.videocardbenchmark.net/high_end_gpus.html) rating around 3000 (GeForce GTX 750) <br/> - 2 GB Video Memory | 4 GB |
 
-## Notes
+#### Recommended 
 
-- You need the [Visual C++ 2015 x86 Redistributables](https://www.microsoft.com/en-us/download/details.aspx?id=48145) for this version to work.   
-Note: Visual C++ 2017 is directly compatible with Visual C++ 2015. While the project is built with Visual C++ 2015, either version will work.
```

Comment:
2017 is still compatible (tho we recommend 2019), keep in mind we also have an appveyor build for 2017.

Reply:
Orphis is now building for VC++ 2019, thus 2019 should be the recommended redist.

Reply:
Hadn't all redists after 2015.someupdate been unified?

Reply:
Yes. 2015, 2017 and 2019 redists are all bundled together now. To avoid confusion, since it is built with 2019, I am using 2019 in the doc. It doesn't make sense to tell Orphis to build with VC++ 2019, and then say "oh but we aren't using 2019 features yet so lets still tell people to use 2017". That just creates confusion when we can say, "built with 2019, runs with 2019", and it's a bundle anyways, so there's no extra questions to ask.